graspy/align/sign_flips.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Microsoft Corporation and contributors.
+# Licensed under the MIT License.
+
+import numpy as np
+
+from .base import BaseAlign
+
+
+class SignFlips(BaseAlign):
+    """
+    This module is used to transform two datasets by flipping the signs of all
+    entries along some dimensions. In particular, it does so in a way that
+    brings these two datasets to the same orthant according to some criteria,
+    computed along each dimension. The two critera currently available are the
+    median and the maximum value in magnitude along each dimension.
+
+    There are two choices for which orthant the datasets end up in: either the
+    first orthant (i.e. with all criteras being positive), which is a default,
+    or and orthant in which one the second dataset is already in.
+
+    Parameters
+    ----------
+        criteria : string, {'median' (default), 'max'}, optional
```

Comment:
```suggestion
        method : string, {'median' (default), 'max'}, optional
```
i don't know if this is better, thoughts?

Reply:
(criteria is plural, im sorry im a jerk

Reply:
(maybe criterion is fine?)

Reply:
hahahah, i realized that it's pluaral and it seemed acceptable to me because you use a criterion on each dimension, but i was really hoping it will sneak by you. so i guess i can say i was testing you.

criterion or method 'en?

Reply:
changed to criterion for now. lmk if u prefer method.